src/calendar/__specs__/month_spec.js

```diff
@@ -220,6 +220,64 @@ describe('Month', () => {
     })
   })
 
+  context('in "range" and "selectedDaysLimit" mode', () => {
```

Comment:
It's two contexts: "when mode is range" and "when selectedDaysLimit is specified". I would nest second one into existing "in range selection mode": https://github.com/toptal/simple-react-calendar/blob/master/src/calendar/__specs__/month_spec.js#L184